src/Form/DatasetSubmissionType.php

```diff
@@ -132,7 +132,7 @@ public function buildForm(FormBuilderInterface $builder, array $options)
             ))
             ->add('suppMethods', Type\TextareaType::class, array(
                 'label' => 'Supplemental Information - Methods',
-                'required' => false,
+                'required' => true,
```

Comment:
We use the same Form type in Dataset-review tool, and the PO's don't want it to be required in the Dataset-review tool

Reply:
I've separated them now, but if you have a suggestion for code-reuse, let me know. It's a painful amount of duplication.